_component_design/tables-nested.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Tables - Nested
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Nested tables are a specialized sub pattern of basic tables. Nested tables come in a variety of forms and allow users to drill into additional data within a table row. By hiding this data behind a user interaction we maintain the low cognitive load goals of the table pattern, while allowing for more data rich layouts.
```

Comment:
Nix the first sentence.  It's not adding much.

Reply:
removed.